
class Error(object):

    def __init__(self):
        pass

    def check(self)
        pass
